README.md

```diff
@@ -1 +1,13 @@
 # Project 3
+
+For Project 3, you will work independently to create
+an application that solves a problem of your choosing.
+
+## Soft Requirements
```

Comment:
Yeah, maybe just 'requirements'? And change the wording in the FAQ portion accordingly...